backend/Api/Consultants/ConsultantController.cs

```diff
@@ -41,14 +44,76 @@ public ConsultantController(ApplicationContext context, IMemoryCache cache)
+    }
+
+    [HttpGet("perf")]
+    public ActionResult<List<ConsultantReadModel>> GetFast()
+    {
+        var watch = Stopwatch.StartNew();
+
+        var weekSet = DateService.GetNextWeeks(new Week(2023, 44), 8);
+
+        var minDate = new DateOnly(2023, 11, 1);
+        var maxDate = new DateOnly(2023, 12, 1);
+
+        var year = weekSet[0].Year;
+        var minWeek = weekSet.Select(w => w.WeekNumber).Min();
+        var maxWeek = weekSet.Select(w => w.WeekNumber).Min();
+
+
+        var consultants = _context.Consultant.Include(consultant => consultant.Department)
+            .ThenInclude(department => department.Organization).ToList();
+
+        var projects = _context.Project.Include(p => p.Customer)
+            .ToDictionary(project => project.Id, project => project);
+        var absences = _context.Absence.ToDictionary(absence => absence.Id, absence => absence);
+
+        var billableStaffings = LoadStaffingByProjectTypeForWeeks(weekSet, ProjectState.Active);
+        var offeredStaffings = LoadStaffingByProjectTypeForWeeks(weekSet, ProjectState.Offer);
+
+        var plannedAbsences = _context.PlannedAbsence
+            .Include(plannedAbsence => plannedAbsence.Consultant)
+            .Include(plannedAbsence => plannedAbsence.Absence)
+            .Where(absence => absence.Year == year && minWeek <= absence.WeekNumber && absence.WeekNumber <= maxWeek)
+            .GroupBy(plannedAbsence =>
+                new StaffingGroupKey(plannedAbsence.Consultant.Id, plannedAbsence.Absence.Id, plannedAbsence.Year,
+                    plannedAbsence.WeekNumber))
+            .ToList();
+
+
+        var vacations = _context.Vacation
+            .Where(vacation => minDate <= vacation.Date && vacation.Date <= maxDate)
+            .Include(vacation => vacation.Consultant)
+            .GroupBy(vacation => vacation.Consultant.Id)
+            .ToList();
+
+        var a = consultants.Select(c =>
+        {
+            var billableSet = billableStaffings.Where(g =>
+                g.Key.ConsultantId == c.Id);
+            var offeredSet = offeredStaffings
+                .Where(g => g.Key.ConsultantId == c.Id);
+            var absenceSet = plannedAbsences.Where(g => g.Key.ConsultantId == c.Id);
+
+            var consultantVacations = vacations.Where(g => g.Key == c.Id).Aggregate(new List<Vacation>(),
+                (list, grouping) => list.Concat(grouping.Select(v => v)).ToList());
+
+            return ConsultantReadModel.FromController(c, projects, absences, billableSet, offeredSet, absenceSet,
+                consultantVacations, weekSet);
+        }).ToList();
+        watch.Stop();
+
+        return Ok(new { Time = watch.ElapsedMilliseconds, Data = a });
     }
 
 
     private List<ConsultantReadModel> GetConsultantsWithAvailability(string orgUrlKey, Week initialWeekNumber,
         int numberOfWeeks)
     {
-        if (numberOfWeeks == 8)
+        if (numberOfWeeks == 8 && false)
```

Comment:
```suggestion
        if (CacheEnable && numberOfWeeks == 8)
```
![boom-mic-drop-gif](https://github.com/varianter/vibes/assets/39614013/f72c8003-3821-4ca7-9bb8-7ad185434fe0)

Reply:
Ups, denne skal helt vekk. Dette var bare for å tvinge vekk bruken av cache for å sammenligne kjøretiden mellom gammel og ny måte